README.md

```diff
@@ -108,29 +85,26 @@ Feel free to contact us directly and discuss opportunities. Best way is either t
 
 ## Open milestones
 
-The following are in no particular order and may or may not be picked up by us
-
- - Support Peerplays blockchain
- - Support BEOS blockchain
- - Support Ethereum blockchain
- - Support Hardware wallets (Ledger and Keybox in the making)
- - Localisation
- - Extra information & features on dashboard as needed
+ - Support other blockchains
+ - Support hardware wallets
+ - Additional locale translations
  - Port to mobile
 
-# Can I trust this code?
+## Encountered an issue? Want a new feature?
 
-    Don't trust. Verify!
+Open a [new issue](https://github.com/bitshares/beet/issues/new/choose) and fill out the template.
 
-We will pursue a code review and audit
-by an external and independent party as soon as our budget allows to show that we take this serious.
-In the meantime we recommend every user to audit and verify any underlying code for its validity and suitability,
-including reviewing any and all of your project's dependencies.
+If you're skilled in Vue, electron or even just want to help localize the wallet, then fork the repo, create a new branch for your idea/task and submit a pull request for peer review.
 
-    DISCLAIMER
+### Donations
 
-Beet is in *BETA* stage and there is no guarantee of a bug free product. Beet is delivered as is in accordance to the MIT license.
-There is no fear of your keys being exposed / compromised.
+If you like what we are doing and want to support us, donations are much welcomed and will be used to further Beet integration, adoption and development. Donations can be done to following
+accounts / addresses
+
+BitShares Blockchain
 
-You are however welcome to try it out, submit bugs and feature requests as an issue.
+    Account Name: beet Id: 1.2.1152309
+
+Bitcoin Blockchain
 
+    Address: 1GquUmypi1mKg696qRsmM24xmgwVqGGMdA
```

Comment:
I guess these donation addresses are still accessible and the funds going there will be used accordingly? Actually, since they are no longer managed by the currently active developers, I suggest that we replace them or simply remove the section.

Reply:
I think removing them is fair, the owner of the accounts has moved onto other projects so we don't have access to these accounts without reaching out.